centersliderview/src/main/res/values/attrs.xml

```diff
@@ -1,18 +1,4 @@
 <?xml version="1.0" encoding="utf-8"?>
 <resources>
-    <!-- attributes for the CenterSliderView -->
```

Comment:
can remove this empty resource file.

Reply:
will do; I guess we can re-add if there's ever a reason to have a private `attrs.xml` file